Makefile

```diff
@@ -38,9 +42,28 @@ test-unit:
 test-functional:
 	pipenv run ./scripts/run_tests_functional.sh
 
+test-functional-headless:
+	EQ_RUN_FUNCTIONAL_TESTS_HEADLESS='True' pipenv run ./scripts/run_tests_functional.sh
+
+test-single-functional:
```

Comment:
I think we should update the readme to specify using the makefile commands where it references running single tests/test suites 👍

Reply:
👍